### DISCUSSION TITLE
Upgraded from Docker 4.14.* to 4.17.0 (Mac) - Unable to resolve dnsmasq hosts

### DISCUSSION THREAD

**Discussion by janssensjelle** [wardenenv/warden#617] [Help & Support] [answered]
Hello

It seems that something changed in Docker with their privileged situation. I can't seem to connect to any of my sites (nor the main .warden.test services) since upgrading to v4.17.0.

Did I do something wrong here? When I downgrade Docker back to 4.14.x it will work. Only way to get it to work on this version is manually editing /etc/hosts

Mac Studio M1 Ultra
macOS Ventura 13.2.1

Docker Desktop 4.17.0
Docker engine 20.10.23
Compose 2.15.1
Credential Helper 0.7.0

thanks!

**Comment by wigman** [comment]
I had some issues as well, on a new mac, but with migrated data from my previous mac. So I thought it had something to do with old Valet+ configuration that I completely wiped on the new machine.

Eventually, I was missing the `/etc/resolver/test` file. It was somehow deleted in the process. Make sure that file still exists, and contains
```nameserver 127.0.0.1```

I think that was the main issue on my machine. It works now, under Docker version 4.17.0, on M2 Max running Verntura.

**Comment by janssensjelle** [comment]
Thanks for the help @wigman , but it seems this file exists and contains the mentioned nameserver 127.0.0.1 :(

**Comment by navarr** [comment]
There's an older issue with a similar problem: https://github.com/wardenenv/warden/issues/569#issuecomment-1374402830

What worked for them was uninstalling/reinstalling, but the problem isn't entirely understood yet.

**Comment by zamoroka-vaimo** [comment]
Have the same issue. Tried complete docker uninstall with all containers and configurations. Still `ping test.test` is not working :( 
`ping: cannot resolve test.test: Unknown host`

**Reply by janssensjelle** [reply]
Same issue here still after following the steps from #569

**Comment by janssensjelle** [comment, marked as answer]
I was able to resolve it somehow. It worked again after I had removed from my Users directory's `~/.warden/.env ` the following line `TRAEFIK_LISTEN=0.0.0.0`

**Reply by zamoroka-vaimo** [reply]
@janssensjelle I don't have the `TRAEFIK_LISTEN` line in `~/.warden/.env` could you please share the content of your file? 
here is my content
```
# Set to "1" to enable global Portainer service
WARDEN_PORTAINER_ENABLE=0
# SEt to "0" to disable DNSMasq
WARDEN_DNSMASQ_ENABLE=1
```

**Reply by janssensjelle** [reply]
Mine is completely empty now. One other thing I did was to do the install of docker via CLI

https://docs.docker.com/desktop/install/mac-install/

**Reply by zamoroka-vaimo** [reply]
still no luck with empty .env file :(
